go_modules/lib/dependabot/go_modules/file_updater/go_mod_updater.rb

```diff
@@ -90,32 +92,29 @@ def update_files # rubocop:disable Metrics/AbcSize
             run_go_vendor
             run_go_mod_tidy
```

Comment:
`go mod tidy` should run before `go mod vendor` to ensure any dependencies removed by `go mod tidy` are also removed from vendors.